src/SuggestionsOverlay.js

```diff
@@ -128,30 +141,41 @@ class SuggestionsOverlay extends Component {
     return <LoadingIndicator style={this.props.style('loadingIndicator')} />
   }
 
-  handleMouseEnter(index, ev) {
+  handleMouseEnter = (index, ev) => {
     if (this.props.onMouseEnter) {
       this.props.onMouseEnter(index)
     }
   }
 
-  select(suggestion, queryInfo) {
+  select = (suggestion, queryInfo) => {
     this.props.onSelect(suggestion, queryInfo)
   }
+
+  setUlElement = el => {
+    this.ulElement = el
+  }
+}
+
+const getID = suggestion => {
+  if (suggestion instanceof String) {
+    return suggestion
+  }
+
+  return suggestion.id
 }
 
-const styled = defaultStyle(({ position }) => ({
+const styled = defaultStyle({
```

Comment:
the `position`, `left`, and `right` css properties are now set as inline styles and not passed through substyle. This is the preferred approach for dynamic styles and is more performant as well.